log.go

```diff
@@ -0,0 +1,14 @@
+package taskmanager
+
+type Logger interface {
+	Tracef(format string, args ...interface{})
```

Comment:
We can remove the interfaces which we dont use. Better to have less. This will reduce the mock code as well.

Currently, we only use Error and Info. So lets just have those 2